src/options/options.js

```diff
@@ -51,31 +49,31 @@ elements.importBookmarksButton.addEventListener('click', () => {
   } else {
     const reader = new FileReader();
     reader.readAsText(file, 'UTF-8');
-    reader.onload = (evt) => {
+    reader.onload = evt => {
       const fileContents = evt.target.result;
-      // TODO: handle JSON parsing errors
-      const bookmarksArray = JSON.parse(fileContents);
-      if (Array.isArray(bookmarksArray)) {
-        if (!bookmarksArray.length > 0) showNotification('No bookmark ids found in file', 'negative', 'snackbar-options');
-        else {
-          updateBookmarks(bookmarksArray);
+      try {
+        const bookmarksArray = JSON.parse(fileContents);
+        if (Array.isArray(bookmarksArray)) {
+          if (!bookmarksArray.length > 0)
+            showNotification('No bookmark ids found in file', 'negative', 'snackbar-options');
+          else {
+            updateBookmarks(bookmarksArray);
+          }
+        } else {
+          showNotification('Contents not in valid format of ["id1", "id2", ...]', 'negative', 'snackbar-options');
         }
-      } else {
-        showNotification(
-          'Contents not in valid format of ["id1", "id2", ...]',
-          'negative',
-          'snackbar-options',
-        );
+      } catch (error) {
+        showNotification('This is not a valid JSON', 'negative', 'snackbar-options');
       }
     };
   }
 });
 
 // tab switching logic
-document.getElementById('vocab-tabbed-header').addEventListener('click', (e) => {
+document.getElementById('vocab-tabbed-header').addEventListener('click', e => {
```

Comment:
You have removed params around arrow functions. This project uses eslint(Airbnb style) for linting and this violates [arrow-params](https://eslint.org/docs/rules/arrow-parens) rule.

Reply:
I will update this format then @makkoncept